### ISSUE TITLE
SVG output can include strangely encoded chars for truncated text fields

### ISSUE THREAD

**Issue by s-andrews** [s-andrews/SeqMonk#17]
We've been sent an example SVG file of a boxplot which gives validation errors against the SVG spec and won't load in illustrator.  The offending region is:

```
<text x="744" y="36" fill="rgb(0,0,0)" font-family="Arial" font-size="13">overlaÉ</text>
```
..where the text in the file is truncated after overla but some odd character seems to be added at the end.  The original report said this was a Mac specific issue, but it might just be the font sizes being different which trigger this.

**Comment by s-andrews** [comment]
So the text which should be there should be an ellipsis which is a literal three dots added by the boxplot view.  I've just tested the same thing on a PC and the corresponding code is:

```
<text x="637" y="32" fill="rgb(0,0,0)" font-family="Arial" font-size="11">overla...</text>
```

So it looks like something is converting these characters into a single ellipsis and this is breaking things.  I did a quick search but can't see any mention of this being a known feature.  I'll have to see if I can reproduce this on a Mac and then figure out which step in the process is doing the conversion.

**Comment by s-andrews** [comment]
I can't reproduce this.  I've now tried the same plot on a mac (OSX sierra, jdk1.8) and I get 3 separate dots in the svg file, not a single ellipsis character.  I wonder if the file might have been through some other program between seqmonk and adobe illustrator which made this change and broke it?

**Comment by s-andrews** [comment]
I managed to get onto the machine where this bug was originally reported and I *could* reproduce it there.

The machine was running java 1.6.0_65 under Yosemite.  I think this was the version of java which apple maintained rather than a Sun/Oracle build which would have had to be used on later machines.

I upgraded the version of java to the latest (1.8.something), and the bug went away, so it's specific to that older version of the JRE.

Since I can't even install that on my Mac, and since this is going to affect only a minority of installs, and there is a trivial fix I'm not going to pursue this any more.
